.github/workflows/integrate.yaml

```diff
@@ -92,6 +97,8 @@ jobs:
             symfony-version: 6
           - php-version: 8.0
             symfony-version: 3
+          - php-version: 8.0
+            symfony-version: 6
```

Comment:
@VincentLanglet Why exclude Symfony6 with PHP8?

Reply:
It's currently a big failure because codeception doesn't support symfony 6.
https://github.com/Codeception/Codeception/blob/4.1/composer.json#L24

I can add back if you prefer, it will just mean the ci will always fail.

Reply:
no problem, once codeception dependency is fixed, we can remove the ignore